app/views/planning_applications/_planning_application_tabs.erb

```diff
@@ -1,22 +1,23 @@
 <div>
   <ul class="govuk-tabs__list" id="planning_applications_statusTab" role="tablist">
-      <% unless exclude_others? && current_user.reviewer? %>
-      <li class="govuk-tabs__list-item govuk-tabs__list-item--selected" role="presentation">
-        <%= link_to "In assessment", "#in_assessment", class: "govuk-tabs__tab", role: "tab", "aria-controls":"in-assessment", "aria-selected": true, tabindex: 0 %>
-      </li>
-    <% end %>
-    <li class="govuk-tabs__list-item" role="presentation">
-      <%= link_to "Awaiting manager's determination", "#awaiting_determination", class: "govuk-tabs__tab", role: "tab", "aria-controls":"awaiting-determination", "aria-selected": false, tabindex: 1 %>
-    </li>
-    <li class="govuk-tabs__list-item" role="presentation">
-      <% if current_user.assessor?%>
-        <%= link_to "Corrections requested (#{@planning_applications.awaiting_correction.count})", "#awaiting_correction", class: "govuk-tabs__tab", role: "tab", "aria-controls":"awaiting-determination", "aria-selected": false, tabindex: 2 %>
-      <% else %>
-        <%= link_to "Corrections requested", "#awaiting_correction", class: "govuk-tabs__tab", role: "tab", "aria-controls":"awaiting-determination", "aria-selected": false, tabindex: 2 %>
+        <% unless current_user.reviewer? && exclude_others? %>
```

Comment:
I'm a bit confused about what `exclude_others?` does here?

Reply:
I was too! I looked it up and it's essentially a filter that allows either the assessor or the reviewer to see all applications that are currently sitting in assessment (the default view is for the assessor to see only their own, and for the reviewer to not see any in assessment planning applications at all). I think we need to refactor this because the new statuses mean it makes less sense.

Reply:
We can refactor at a later date.